### DISCUSSION TITLE
Wish I could increase the size of toolbar buttons.

### DISCUSSION THREAD

**Discussion by HubKing** [notepadqq/notepadqq#1085] [Ideas]
The buttons seems quite small. There currently is no way to increase the size of toolbar buttons, right? If so, I wish that becomes possible. If I could remember correctly, Notepad++ allows making toolbar icons bigger.

![image](https://user-images.githubusercontent.com/10900489/229327720-38288fbd-b7fe-46db-ad49-c7e4f9a34d16.png)

**Comment by ChooChooAl** [comment]
Me too.
